src/main/resources/assets/farmersdelight/lang/en_us.json

```diff
@@ -249,7 +249,7 @@
   "farmersdelight.block.cutting_board.invalid_item": "This doesn't seem cuttable...",
   "farmersdelight.block.cutting_board.invalid_tool": "Maybe with a different tool...",
   "farmersdelight.block.skillet.invalid_item": "This can't be cooked...",
-  "farmersdelight.item.skillet.how_to_cook": "Use your other hand to hold ingredients!",
```

Comment:
The use of "other" was intended in this case, as the Skillet can be operated from either hand. Can you please revert the change?